add_clip_ui.ui

```diff
@@ -43,11 +43,14 @@
    </property>
   </widget>
   <widget class="QRadioButton" name="newButton">
+   <property name="enabled">
+    <bool>true</bool>
```

Comment:
This has no effect because widgets are enable by default. Maybe you want it checked by default ? use "checked" property for that.